File: greetings2.c
#include <stdio.h>
int main() {
    char a[1000]; // {e,e,m,k,j,h}
    scanf("%s",&a);
   

    for(int i=0;i<100;i++) {
        if(a[i]=='L' || a[i]=='l') {
            printf("Alligator!");
            break;
        }
        if(a[i]=='w' || a[i]=='W') {
            printf("what's uhat's up");
            break;
        }
        
        if(a[i]=='h' && a[i+1]=='e') {
            // for(int i=0;i<2;i++) {
            //     printf("e");
            // }

            printf("ee");
        }
        else {
            printf("%c",a[i]);
            if(a[i]=='y') {
                break;
            }
        }
    }
}

File: Greetings2.java
import java.util.Scanner;

public class Greetings2 {
    public static void main(String[] args) {
        Scanner keyboard = new Scanner(System.in);
        String input = keyboard.nextLine();
        int panjang = input.length();
        int jumlah = 0;
        if(input.charAt(panjang-1)=='y') {
            for(int i = 0; i<input.length(); i++) {
                if(input.charAt(i)=='e') {
                    jumlah++;
                }
            }
            for(int i = 0; i<1; i++) {
                System.out.printf("h");
                for(int j = 0; j<jumlah; j++) {
                    System.out.printf("ee");
                }
                System.out.printf("y\n");
            }
        }
        else if(input.equals("what's up")) {
            System.out.printf("what's uhat's up\n");
        }
        else if(input.equals("Later!")) {
            System.out.printf("Alligator!\n");
        }
    }
}
